### DISCUSSION TITLE
Отображение значения таблицы и поля с селектом, кодом селект в таблице циклов

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#306] [Как это сделать?] [answered]
Здравствуйте подскажите пожалуйста, как взять значение селекта из таблицы в другую таблицу при автозаполнении? Например есть таблица договоры, в ней поле заказчик заполняется селектом из таблицы заказчики, а в таблице циклов, где я хочу и почти сделал автозаполнение всех полей договора по названию договора.(код: select(table: 'dogovory'; field: 'zakazchik_polnoe'; where: 'id' = #f_dogovor). Т.е получаеться селект на селект и получаю число в конечной таблице, возможно ли получать названия, или как то иначе стоит это сделать?

**Comment by erpic-ru** [comment, marked as answer]
А в коде селекта-то (selectRowListForSelect) что находится? 
Если хочется получать названия в строчное поле, то следует использовать sfield: select(table: 'dogovory'; sfield: 'zakazchik_polnoe'; where: 'id' = #f_dogovor)

**Reply by ttm-alexeypolunin** [reply]
Вот по этой теме еще: https://ru.docs.totum.online/how-do-I-take-the-value-of-a-select

**Reply by ttm-alexeypolunin** [reply]
И еще если вы в конечной таблице хотите Селект, то он должен быть не по селекту, а по таблице источнику первого селекта. Вот здесь в середине это разбирается: https://ru.docs.totum.online/crossed-out-selects-and-what-it-may-be-related-to

**Comment by Leo-ui-528** [comment]
Селект был такой selectRowListForSelect(table: 'zakazchiki'; field: 'nimenovanie'; order: 'id' asc;), получилось, спасибо

**Comment by mihail-cornev** [comment]
А я делаю поля с типом Селект и потом селектами вытаскиваю нужные значение. Тут удобно, что само значение является id-шником всегда и можно потом и в других местах использовать селекты с селектами )
Получается что я всегда вижу отображаемое значение которое мне нужно, хотя основание всегда id.
![Screenshot_10](https://user-images.githubusercontent.com/113771444/211614431-2d162a9a-0e63-4c16-98ba-4631a072ff36.png)
